internal/server/models/deviceflowauthrequest.go

```diff
@@ -8,13 +8,20 @@ import (
 
 type DeviceFlowAuthRequest struct {
 	Model
-	UserCode       string
-	DeviceCode     string
-	Approved       *bool
-	AccessKeyID    uid.ID
-	AccessKeyToken string // to be filled in once approved
-	ExpiresAt      time.Time
+	UserCode   string
+	DeviceCode string
 
-	// can be preloaded if there's an AccessKeyID
-	AccessKey *AccessKey
+	// TODO: remove Approved field? There's no way to reject a request right now
+	// and AccessKeyID != nil indicates approved.
+	Approved    *bool
+	AccessKeyID uid.ID
+
+	// AccessKeyToken is set once the request is approved.
+	// TODO: use EncryptedAtRest
+	AccessKeyToken string
+
+	ExpiresAt time.Time
+
+	// AccessKey will be populated by some queries, but is never used on writes.
+	AccessKey *AccessKey `db:"-"`
```

Comment:
`db:"-"` is a gorm thing, right?

Reply:
Nope, Gorm uses `gorm` as the field tag. This is the field tag we're using for generating the table methods in `TestGenerateTableMethods`.  I added because it can be a good indicator that this field is not always set, but I'm happy to omit it if we don't plan on using the method generation on this type.